eng/mgmt/mgmtmetadata/resources_resource-manager.txt

```diff
@@ -12,3 +12,15 @@ Commit:      d547b7794cad660fd3cceee790e407a4d534308c
 AutoRest information
 Requested version: latest
 Bootstrapper version:    autorest@2.0.4283
+
```

Comment:
Why do you have changes to this file and the reservations metadata file at the end?

Reply:
this looks like a mistake

Reply:
Sorry for the mistake. revert the change